homeassistant/components/xbox/media_source.py

```diff
@@ -0,0 +1,282 @@
+"""Netatmo Media Source Implementation."""
+from dataclasses import dataclass
+import logging
+from typing import List, Tuple
+
+# pylint: disable=no-name-in-module
```

Comment:
This was needed due to Cython not loading it properly - see http://pylint.pycqa.org/en/latest/technical_reference/c_extensions.html